### PR TITLE
Feature/delete user

### PR DESCRIPTION
**Pull request by hammad-Ikhlaq-7vals**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -34,6 +34,19 @@ def update
       end
   end
 
+  def destroy
+    binding.pry
```
Comment (mujeeb-arshad) [inline app/controllers/users_controller.rb]: again :/

File: app/models/ability.rb [hunk 2]
```diff
@@ -29,7 +29,8 @@ def initialize(user)
 
     # *** ADMINS and OWNERS ***
     can :manage, :all, company_id: user.company_id
-    cannot :destroy, User, role: User::ROLES[2]
+    cannot :destroy, User, company_id: user.company_id, role: User::ROLES[2]
```
Comment (mujeeb-arshad) [inline app/models/ability.rb:33]: why is it required? "cannot"

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (hammad-Ikhlaq-7vals) [reply app/controllers/users_controller.rb]: oh

File: app/models/ability.rb (hunk 2 above)
Comment (hammad-Ikhlaq-7vals) [reply app/models/ability.rb:33]: Buttons do not disappear when use only "can"

Comment (hammad-Ikhlaq-7vals) [reply app/models/ability.rb:33]: There was one extra can. I have removed it.

**Pull request merged**